third_party/gfootball_engine/src/systems/graphics/rendering/sdl_glfuncs.h

```diff
@@ -188,6 +188,7 @@ SDL_PROC_UNUSED(void,glFrontFace,(GLenum mode))
 SDL_PROC_UNUSED(void,glFrustum,(GLdouble left, GLdouble right, GLdouble bottom, GLdouble top, GLdouble zNear, GLdouble zFar))
 SDL_PROC_UNUSED(GLuint,glGenLists,(GLsizei range))
 SDL_PROC(void,glGenBuffers,(GLsizei n, GLuint * buffers))
+SDL_PROC(void,glGenerateMipmap,(GLenum target))
```

Comment:
The function is present since OpenGL 3.0:
https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glGenerateMipmap.xhtml